modules/cmdb/conf/conf.go

```diff
@@ -500,3 +505,21 @@ func EnableNS() bool {
 func QAAddr() string {
 	return cfg.QAAddr
 }
+
+func OryEnabled() bool {
+	return cfg.OryEnabled == "true" || cfg.OryEnabled == "1"
+}
+
+func OryKratosAddr() string {
+	return cfg.OryKratosAddr
+}
+
+func OryKratosPrivateAddr() string {
+	if cfg.OryKratosPrivateAddr != "" {
+		return cfg.OryKratosPrivateAddr
+	} else if cfg.OryKratosAddr != "" {
+		// TODO: ugly hack!
+		return cfg.OryKratosAddr[:len(cfg.OryKratosAddr)-1] + "4"
```

Comment:
please add comment and examples for this ugly hack.

Reply:
rollback to simple config, no hack